### PR TITLE
Update release notes and Prometheus support page

### PR DESCRIPTION
**Pull request by stevebang**

### CODE REVIEW

File: content/influxdb/v1.7/about_the_project/releasenotes-changelog.md [hunk 1]
```diff
@@ -7,22 +7,47 @@ menu:
...
+
+<dt>
+If your InfluxDB OSS server is using the default in-memory index (`[data].index-version = "inmem"`),
+this release includes the fix for InfluxDB 1.7.5 servers that stopped responding without warning. 
+</dt>
+
+### Features
+
+- Upgrade Flux to `0.24.0` and remove the platform dependency.
+  - If Flux is enabled, use Chronograf 1.7.9 or later.
```
Comment (timhallinflux) [inline content/influxdb/v1.7/about_the_project/releasenotes-changelog.md]: this should be Chronograf 1.7.10

File: content/influxdb/v1.7/about_the_project/releasenotes-changelog.md [hunk 2]
```diff
@@ -7,22 +7,47 @@ menu:
...
 ## v1.7.5 [2019-03-26]
--------------------
 
 <dt>
 **Update (2019-04-01):** If your InfluxDB OSS server is using the default in-memory index (`[data].index-version = "inmem"`), then do not upgrade to this release. Customers have reported that InfluxDB 1.7.5 stops responding without warning. For details, see [GitHub issue #13010](https://github.com/influxdata/influxdb/issues/13010). The [planned fix](https://github.com/influxdata/influxdb/issues/13053) will be available soon.
 </dt> 
 
+### Features
+
+- Add `gen-init` and `gen-exec` commands to `influx_tools` utility (internal use only). Add support for describing data-generation schema using a TOML file.
```
Comment (timhallinflux) [inline content/influxdb/v1.7/about_the_project/releasenotes-changelog.md]: what does this mean?  if these are "internal use only" why are we describing these?

We should check the official `influx_tools` package to ensure that these are actually in....

File: content/influxdb/v1.7/about_the_project/releasenotes-changelog.md (hunk 1 above)
Comment (stevebang) [reply content/influxdb/v1.7/about_the_project/releasenotes-changelog.md]: thanks, i was waiting for a version from Russ

File: content/influxdb/v1.7/about_the_project/releasenotes-changelog.md (hunk 2 above)
Comment (stevebang) [reply content/influxdb/v1.7/about_the_project/releasenotes-changelog.md]: Removed -- figured if it was in an OSS release, it's not hidden. And, it's in the changelog for the OSS release too. So, I added the "(internal use only)".

**Pull request merged**